api/src/test/java/io/druid/data/input/impl/TimestampSpecTest.java

```diff
@@ -37,6 +38,17 @@ public void testExtractTimestamp() throws Exception
   }
 
   @Test
+  public void testExtractTimestampWithTimezone() throws Exception
+  {
+    DateTimeZone eventTimeZone = DateTimeZone.forID("Asia/Shanghai");
+    TimestampSpec spec = new TimestampSpec("TIMEstamp", "yyyy-MM-dd HH:mm:ss.SSS", null, eventTimeZone.toString());
```

Comment:
`eventTimeZone` seems redundant here, why not just pass `"Asia/Shanghai"` as a string directly?